packs/mysql/config.yml

```diff
@@ -4,8 +4,31 @@ name: mysql
 
 # Description of the pack
 description: |+
-  Official New Relic pack for MySQL and MariaDB
-  Use this pack together with the New Relic MySQL On Host Integration to get insight into the performance of your MySQL and MariaDB instances.
+  # MySQL Performance Monitoring
+
+  Applications powered by relational database management systems demand the user to understand how the application uses it. Quickly identify and resolve the source server issues with MySQL performance monitoring tools. 
```

Comment:
@bradleycamacho tagged. I feel these are too wordy....  they read like marketing content but if this is the approach we are taking with these descriptions for SEO reasons, i'll stop commenting. 😄

Reply:
I cut down the content for clarity.